File: Codigo Java/src/Cliente.java

package gestorsupermercado;

/**
 *
 * @author LexV
 */
public class Cliente {
    private String nombre;
    private String apellido;
    private int id;

    public Cliente(String nombre, String apellido, int id) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public int getId() {
        return id;
    }
}

File: cliente.py
# cliente.py

class Cliente:
    def __init__(self, nombre, apellido, id_cliente):
        self.nombre = nombre
        self.apellido = apellido
        self.id_cliente = id_cliente
    
    def mostrar_info(self):
        return f"Nombre: {self.nombre}, Apellido: {self.apellido}, ID Cliente: {self.id_cliente}"
